CHANGELOG.adoc

```diff
@@ -12,6 +12,7 @@
 === Breaking changes
 
 - https://github.com/eclipse-sirius/sirius-components/issues/1088[#1088] [core] Change the type of `IRepresentationDescription#id` from UUID to String. This will allow us, when we will receive an operation to perform with a given representation description identifier, to determine if this operation should be handled by the Sirius Desktop compatibility layer, the View support or by a programmatic API
+- https://github.com/eclipse-sirius/sirius-components/issues/689[#689] [core] Implementations of `IObjectService` must now implement `getObjects` in addition of current `getObject`
```

Comment:
Why? I told you that you could do that in your own custom implementation of ObjectService, I don't see why it needs to be an API break for everybody else.